packages/customer/src/migrations/Migration20240124154000.ts

```diff
@@ -0,0 +1,79 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20240124154000 extends Migration {
+  async up(): Promise<void> {
+    // Adjust existing 'customer' table
+    this.addSql(
```

Comment:
q: in this migration, do we need to take into account to drop unnecessary columns from v1 and their index? or do we plan to keep the existing columns and just alter/add the changed columns and then do the full migration separately? for the data, should we include that in the migration as well? e.g moving the billing_address to default_belling_address or things like that if necessary, I took this column as example, it is not necessarly true? what is the global plan?

Reply:
It's a good question for now I am working with the assumption that we only get tables "brought up to speed" with modules. Eventually we will have to figure out a plan for data migration, but I am not entirely sure if this should be the responsibility of the module - right now I am leaning more toward this being something the core handles (potentially with a workflow :))

Reply:
yes indeed, we could do something like that, indeed and that way it can work no matter where the database are in both parties 🎉